doc/developer/workflow.rst

```diff
@@ -108,6 +108,34 @@ March/July/November.  Walking backwards from this date:
    (The :file:`configure.ac` edit and tag push are considered git housekeeping
    and are pushed directly to ``master``, not through a PR.)
 
+   Below is the snippet of the commands to use in this step.
+
+     .. code-block:: console
+
+        % git remote --verbose
+        origin    git@github.com:ton31337/frr (fetch)
+        origin    git@github.com:ton31337/frr (push)
+        upstream  https://github.com/frrouting/frr (fetch)
+        upstream  https://github.com/frrouting/frr (push)
+
+        % git checkout master
+        % git pull upstream master
+        % git checkout -b dev/8.2
+        % git tag base_8.2
+        % git tag frr-8.2-rc1
```

Comment:
I think we agreed to drop rc1 and just have the `base` tag, right?

Reply:
Updated.